source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -144,6 +157,8 @@ private void preview(){
     private void setupPaintCollection() {
 
         List<Document> filterdata = new ArrayList<>();
+
+        // Set up the "vendor" field for your documents
```

Comment:
Suggestion:
I think this should be omitted since it's setup code meant only for tech writers